status-im/status-react
# Refactor app db

# Steps to test

This is a naming refactoring so there shouldn't be any functionality changes. Requires a sanity check only
Check that updates from current version of app works, contacts and messages are not lost and are displayed as normal.

status: ready
